### ISSUE TITLE
Get the chapter (number or name) in the export

### ISSUE THREAD

**Issue by dcaixinha** [pettarin/export-kobo#5]
Hi! First of all, thanks for this amazing tool. It's really helpful to export notes — especially for technical books! However, most of the times I want to see the chapter where I highlighted some text. Is this information available for extraction in the sqlite DB?

I can work on submitting a PR for this @pettarin, but I'd really love some pointers on where is this info possible and so on (or whether this is possible or not). Really appreciate any help! Thanks 🙌

**Comment by pettarin** [comment]
Hi,

unfortunately I no longer own a Kobo device.

You might want to open (a copy of) the KoboReader.sqlite file from your 
device, with an SQLite reader application like SQLitebrowser.

The last sqlite file I have (which was provided by another user), has 
the following interesting fields:

- table Bookmark : BookmarkID, VolumeID, ContentID --- where ContentID 
is VolumeID + something like "!OEBPS!ch02.html#TCP_HANDSHAKE" which is 
the file inside the EPUB file, roughly equivalent to a chapter

- table content : ContentID (see above), BookID, BookTitle, Title --- 
however, in the specific case of the above example, it seems there are 
several records, viz. one for each chapter or section of the ebook. For 
example, there is a record with ContentID=..."OEBPS!ch02.html" with the 
not-so-great value Title="ch02.html", but also one with

ContentID=..."!OEBPS!ch02.html#TCP_HANDSHAKE-3"

with

Title="Three-Way Handshake"

which might be better. Note, however, that I do not know what the "-3" 
suffix stands for.


In summary, it seems that the Kobo software stores some form of 
information about the EPUB (and PDF) structure, and attempts at 
labelling chapters (perhaps using the EPUB TOC labels), but I do not 
know for sure.

AP
